lib/reducers/create-otp-reducer.js

```diff
@@ -736,7 +736,6 @@ function createOtpReducer(config) {
           // If setting to a route (not null), also set main panel.
           return update(state, {
             ui: {
-              mainPanelContent: { $set: MainPanelContent.ROUTE_VIEWER },
```

Comment:
I am suspecting that this change is interfering with navigating with the back button. To see the effect:
1. Open the route viewer, select a route, and go to pattern view.
2. Click the browser Back button.
3. Observe that the route viewer returns to a state where no route is selected. The route selected in step 1 should remain selected.

Reply:
Great catch thank you! Fixed in [0f8aea1](https://github.com/opentripplanner/otp-react-redux/pull/937/commits/0f8aea157043848a60621d927701606d32f012ca)

Reply:
Great, much better now!